eahub/base/static/global/styles/components/_navbar.scss

```diff
@@ -0,0 +1,105 @@
+@import 'eahub/base/static/global/styles/bootstrap-variables';
+
+$spacing-lg: 25px;
+$spacing-md: 20px;
+$spacing-sm: 15px;
+$spacing-xs: 10px;
+$spacing-xxs: 5px;
+
+.navbar {
+    .navbar-brand {
+        display: flex;
+        align-items: center;
+        
+        .navbar__logo {
+            display: flex;
+        }
+        
+        .navbar__page-name {
+            display: flex;
+            margin-left: $spacing-xxs;
+            color: white;
+            font-size: 1.3rem;
+        }
+    }
+
+    .navbar-collapse {
+        .svg-inline--fa {
+            min-width: $spacing-md;
+            @include media-breakpoint-down(lg) {
+                margin-left: -2px;
+            }
+        }
+
+        .fa-user-plus {
+            @include media-breakpoint-down(lg) {
+                margin-left: -1px;
+            }
+        }
+
+        .nav-link {
+            margin: 0 $spacing-xxs;
+            @include media-breakpoint-down(lg) {
+                margin: 0;
+            }
+        }
+
+        .nav-item.dropdown {
+            .navbar-toggler {
+                height: 37px;
+                align-items: center;
+                margin-left: $spacing-xs;
+                color: transparent;
+                @include media-breakpoint-down(lg) {
+                    margin-left: 0;
+                    margin-bottom: $spacing-sm - 8px;
+                }
+
+                .navbar-toggler-icon {
+                    background-image: url('~global/images/angle-down-solid.svg');
+                    width: $spacing-sm;
+                    height: $spacing-sm;
+                }
+            }
+
+            .navbar-toggler.show {
+                .navbar-toggler-icon {
+                    transform: rotate(180deg);
+                }
+            }
+        }
+
+        .dropdown-menu {
+            right: 0;
+            left: initial;
+            $navbar-height: 45px;
+            top: $navbar-height;
+            box-shadow: 0 2px 6px rgba(0, 0, 0, 0.2), 0 5px 15px rgba(0, 0, 0, 0.1);
+            &.show {
+                @include media-breakpoint-down(lg) {
+                    margin-bottom: $spacing-lg - 8px;
+                }
+            }
+
+            .dropdown-item {
+                margin: $spacing-xs 0;
```

Comment:
@anastasiia-what-digital, I noticed this one here, is there a chance to use padding following the original bs5 style? Margin kind of works, but it decreases the clickable space of course

Reply:
Fixed it.